### PR TITLE
[MOO-1004]: Update widgets icons and structure preview

### PR DESCRIPTION
**Pull request by UrazAkgultan**
## Checklist

-   Contains unit tests ❌
-   Contains breaking changes ❌
-   Compatible with: MX 🔟
-   Did you update version and changelog? ✅ 
-   PR title properly formatted (`[XX-000]: description`)? ✅

#### Feature specific

-   Comply with designs ✅ 

## What is the purpose of this PR?

Icons and the structure preview was updated.

### CODE REVIEW

**Comment by UrazAkgultan** [comment]
> @UrazAkgultan please be aware some icons might change due to https://mendix.slack.com/archives/C8AF2EVEW/p1681742421242449


https://github.com/mendix/native-widgets/pull/40#issuecomment-1511512459

File: packages/tools/piw-utils-internal/src/constants/Colors.ts [hunk 1]
```diff
@@ -79,15 +85,17 @@ export const ColorsLight: ColorsType = {
 export const ColorsDark: ColorsType = {
     background: {
         topBar: {
-            data: "#C2E2FF",
-            standard: "#323232"
+            data: colorWithOpacity("#3A65E5", 20),
+            standard: colorWithOpacity("#646464", 20)
         },
```
Comment (UrazAkgultan) [inline packages/tools/piw-utils-internal/src/constants/Colors.ts]: It won't work if you use colors that have opacity.

**Review by UrazAkgultan — COMMENTED** [review]
@stelselim There are some prettier errors and also I wasn't able to build a widget, build failed due to rollup error.

File: packages/tools/piw-utils-internal/src/constants/Colors.ts (hunk 1 above)
Comment (stelselim) [reply packages/tools/piw-utils-internal/src/constants/Colors.ts]: Reverted back to only color.

**Comment by stelselim** [comment]
> @stelselim There are some prettier errors and also I wasn't able to build a widget, build failed due to rollup error.

Prettier errors should be fixed with last commit.

File: packages/pluggableWidgets/badge-native/src/Badge.editorConfig.ts [hunk 2]
```diff
@@ -4,10 +4,10 @@ import { BadgePreviewProps } from "../typings/BadgeProps";
 import badgeSvgDark from "./assets/Badge.dark.svg";
 import badgeSvgLight from "./assets/Badge.light.svg";
 
-export function getPreview(_: BadgePreviewProps, isDarkMode: boolean): StructurePreviewProps {
+export const getPreview = (_: BadgePreviewProps, isDarkMode: boolean): StructurePreviewProps => {
```
Comment (Ahmad-Elsayed) [inline packages/pluggableWidgets/badge-native/src/Badge.editorConfig.ts:7]: I am curious, what is the reason for this change?

File: packages/pluggableWidgets/bottom-sheet-native/src/BottomSheet.editorConfig.ts [hunk 3]
```diff
@@ -9,85 +9,64 @@ import {
...
-                : [
-                      {
-                          type: "DropZone",
-                          property: values.smallContent,
-                          placeholder: "Always visible"
-                      },
+export const getPreview = (values: BottomSheetPreviewProps, isDarkMode: boolean): StructurePreviewProps => {
+    const content: StructurePreviewProps[] = (
+        values.type === "modal"
+            ? values.modalRendering === "custom"
```
Comment (Ahmad-Elsayed) [inline packages/pluggableWidgets/bottom-sheet-native/src/BottomSheet.editorConfig.ts:15]: It is better to avoid nested ternaries as they make it harder to follow the code

File: packages/jsActions/mobile-resources-native/package.json [hunk 4]
```diff
@@ -9,12 +9,12 @@
...
   "marketplace": {
-    "minimumMXVersion": "10.0.0-beta.2",
+    "minimumMXVersion": "10.0.0.8352",
     "marketplaceId": 109513
   },
   "testProject": {
     "githubUrl": "https://github.com/mendix/native-mobile-resources",
-    "branchName": "main"
+    "branchName": "moo/feat/js-actions-new-icons"
   },
```
Comment (UrazAkgultan) [inline packages/jsActions/mobile-resources-native/package.json:18]: Change `branchName` to main before merge this PR

File: packages/jsActions/nanoflow-actions-native/package.json [hunk 5]
```diff
@@ -9,12 +9,12 @@
...
-    "minimumMXVersion": "10.0.0-beta.2",
+    "minimumMXVersion": "10.0.0.8352",
     "marketplaceId": 109515
   },
   "testProject": {
     "githubUrl": "https://github.com/mendix/native-mobile-resources",
-    "branchName": "main"
+    "branchName": "moo/feat/js-actions-new-icons"
   },
   "scripts": {
```
Comment (UrazAkgultan) [inline packages/jsActions/nanoflow-actions-native/package.json:19]: Change `branchName` to main before merge this PR
